src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -99,6 +98,12 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
             {
                 this.meter.CreateObservableCounter($"{metricPrefix}assembly.count", () => (long)AppDomain.CurrentDomain.GetAssemblies().Length, description: "Number of Assemblies Loaded.");
             }
+
+            if (options.IsExceptionsEnabled)
+            {
+                AppDomain.CurrentDomain.FirstChanceException += (source, e) => Interlocked.Increment(ref this.exceptionCount);
```

Comment:
Use Counter API here, rather than doing the aggregation here and then expose the data via ObservableCounter API.

Reply:
Done.